README.md

```diff
@@ -1,75 +1,70 @@
-# math magicia
-math magicians is an application for math fans, a calculator app is also intgerated for users to make arithmetic calculations.
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
Great job @amklass11 🤝

- [ ] As stated by the previous reviewer, kindly use the correct [readme template](https://github.com/microverseinc/readme-template) and customize it accordingly. Remember to fix the MIT license link which is broken.